src/injectHighlightWrappers.ts

```diff
@@ -16,8 +16,14 @@ const NODE_TYPE = {
  */
 const IGNORE_TAGS = [
   'SCRIPT', 'STYLE', 'SELECT', 'OPTION', 'BUTTON', 'OBJECT', 'APPLET',
-  'VIDEO', 'AUDIO', 'CANVAS', 'EMBED', 'PARAM', 'METER', 'PROGRESS',
+  'AUDIO', 'CANVAS', 'EMBED', 'PARAM', 'METER', 'PROGRESS',
 ];
+/**
+ * Highlights can be created around these block and text elements.
+ */
+const BLOCK_ELEMENTS = ['img', 'video', 'iframe'];
```

Comment:
as far as i can tell the videos are also iframes. eg: https://rex-web.herokuapp.com/books/college-physics/pages/2-introduction-to-one-dimensional-kinematics so i'm not sure you have to handle them separately

Reply:
i ran a search and found no books with actual `video` elements